### DISCUSSION TITLE
next build failing because of TS errors on storybook files?

### DISCUSSION THREAD

**Discussion by hems** [vercel/next.js#21045] [Help]
Today I realised one of my deploys failed because of an error on a Storybook file.

Basically, I ran `next build` and got the following error:

`Type error: 'Box' is declared but its value is never read.` on my file named `Accordion.stories.tsx`

What's the recommended way to tell `next build` to ignore story files? Shall I simply add `**/*.stories.tsx` to my `tsconfig.json` under the `exclude` property?

Or... Is there a way to tell `next build` to don't be so strict when building, for instance, only give warnings for minor issues such as a variable not being used but still build the project?

**Comment by alifarooq0** [comment]
Running into this issue. Any ideas here?

**Comment by kristian-nst** [comment]
Did anyone find any solutions?
